pkg/model/serializer.go

```diff
@@ -716,7 +716,7 @@ func (d *ManifestDevs) UnmarshalYAML(unmarshal func(interface{}) error) error {
 }
 
 func isManifestFieldNotFound(err error) bool {
-	manifestFields := []string{"devs", "name", "icon", "variables", "deploy", "destroy"}
+	manifestFields := []string{"devs", "name", "icon", "variables", "deploy", "destroy", "build"}
```

Comment:
@jLopezbarb based on the spec, it should be `dev`, not `devs`:
https://www.notion.so/okteto/Okteto-DEV-915b72bf1a8f4275879974e544a2d614#5ee9c33a37484727a72f5607912f6bfb